pandas/core/strings/base.py

```diff
@@ -8,6 +8,8 @@
 
 from pandas._typing import Scalar
 
+from pandas.core.series import Series
```

Comment:
import of Series is added just for the type annotations and causing the circular import?

try
```
from typing import TYPE_CHECKING
...
if TYPE_CHECKING:
    from pandas import Series
```